src/App.css

```diff
@@ -26,3 +26,82 @@
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+input[type=text], input[type=tel], input[type=email], select {
+    width: 100%;
+    padding: 12px;
+    margin: 8px 0;
+    display: inline-block;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+}
+
+input[type=submit],input[type=button] {
+    width: 100%;
+    background-color: #f50b0b;
+    color: white;
+    padding: 14px 20px;
+    margin: 8px 0;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+}
+
+input[type=button] {
+  background-color: transparent;
+  color:#f50b0b;
+  border: 1px solid #f50b0b;
+}
+
+input[type=submit]:hover {
```

Comment:
Create button classes and apply on the buttons rather than applying styles on native elements. This would make your code more modular, scalable.

Reply:
yes.. we do the same in a project.